tests/common/support/kueue.go

```diff
@@ -129,3 +129,12 @@ func KueueWorkloadAdmitted(workload *kueuev1beta1.Workload) bool {
 	}
 	return false
 }
+
+func KueueWorkloadEvicted(workload *kueuev1beta1.Workload) bool {
+	for _, v := range workload.Status.Conditions {
+		if v.Type == "Evicted" && v.Status == "True" && v.Reason == "Deactivated" {
```

Comment:
```suggestion
		if v.Type == "Evicted" && v.Status == "True" {
```
There are multiple eviction reasons, I afraid this check won't address every one of them : 
Preempted, ClusterQueueStopped, PodsReadyTimeout, InactiveWorkload